package_template.json

```diff
@@ -36,7 +36,7 @@
         "@rollup/plugin-commonjs": "11.0.2",
         "@rollup/plugin-node-resolve": "^7.0.0",
         "@rollup/plugin-replace": "^2.2.0",
-        "rollup": "^2.0.0",
+        "rollup": "^2.0.1",
```

Comment:
do you actually have to change this? `npm` will grab the latest version available. Is there some reason 2.0.0 would be incompatible? that's the only reason to bump it

Reply:
fixed